### PR TITLE
Added a new input plugin to check SSL certs

### PR DESCRIPTION
**Pull request by egarbi**
### Required for all PRs:
- [ ] CHANGELOG.md updated (we recommend not updating this until the PR has been approved by a maintainer)
- [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)
- [x] README.md updated (if adding a new plugin)

### CODE REVIEW

**Comment by wcgcoder** [comment]
The README.md in this pull request for the plugin is incorrect. The example text for telegraf.conf should read:

```
# SSL request given a list of servers (server:port) and a timeout
[[inputs.check_ssl]]
  ## Servers ( Default [] )
  servers = ["github.com:443"]
  timeout = "5s"
```
... in order to fix the lack of quotes and the incorrect variable `response_timeout`.

I'd love to see this plugin in the main product! Please expedite! 👍

**Comment by egarbi** [comment]
@wcgcoder Thanks for your input, Readme has been fixed however checks are failing for an unrelated issue.

**Comment by wcgcoder** [comment]
Thanks so much. Looks like a problem from outside your plugin. Not sure why as I'm rather new to this, but I was able to manually add your plugin to commit `31a4f03` and build it successfully.

**Comment by okv** [comment]
Hi there.
Any news about this? I also need this functionality.

**Comment by danielnelson** [comment]
@okv We are using the github milestones to prioritize pull requests.  When there is new information it will be reflected here.

**Comment by okv** [comment]
will be waiting for it, thanks for your information

**Comment by mcfedr** [comment]
Maybe a separate tool, but how nice would it be to integrate your [SSL Labs](https://www.ssllabs.com/ssltest/) score into this check? https://github.com/ssllabs/ssllabs-scan

**Comment by hardinw** [comment]
@danielnelson I know you said updates will be posted here, but i'm getting itchy to have this functionality in the official release.  Any update on a potential release date?

**Comment by swestcott** [comment]
I've patched the error handling issue on a new branch (fe7235c4d100f369d60c44d71bc602d702180c42) and signed the CLA. Does someone want to cherry-pick this or should I raise a new PR? It could be great to get this released with v1.4.

**Comment by danielnelson** [comment]
@egarbi Can you integrate the fix by @swestcott into your branch?

File: plugins/inputs/ssl/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Check SSL Input Plugin
```
Comment (egarbi) [inline plugins/inputs/ssl/README.md]: So, what exactly should I change? Everything? the directory, file names and the references to them?

Comment (danielnelson) [inline plugins/inputs/ssl/README.md]: Yeah, I think we need to change all of this.

**Comment by antoinealb** [comment]
Since this PR is pretty stalled, what would be the best way to try to get it merged again ? Pick the patches, apply requested changes and open a new PR ?

**Comment by egarbi** [comment]
@antoinealb  if you want to start from where I left, be my guest. This was my first go project ever and I don't really now have much time to spend on this.
Thanks

**Comment by russorat** [comment]
sounds like we should close this in favor of #3768 . Thoughts?

**Pull request closed without merging**